.github/workflows/rust.yml

```
@@ -120,3 +120,29 @@ jobs:
         uses: softprops/action-gh-release@v1
         with:
           files: ./artifacts/*.tar.gz
+
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)